tests/integration/policy/policy_pack_v0-4-1-dev/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
wondering if it makes more sense to call this dir `policy_pack_w_config` since the version is probably meaningless to most people?

Reply:
Agreed, that's a more meaningful name. will update